client/web/src/routes.constants.ts

```diff
@@ -48,6 +48,8 @@ export enum PageRoutes {
     CodySubscription = '/cody/subscription',
 
     CodyManageTeam = '/cody/team/manage',
+    // Accepts an invite to join a Cody team, then redirects to the Cody team page.
+    CodyAcceptInvite = '/cody/invites/accept',
```

Comment:
```suggestion
    CodyAcceptInvite = '/cody/invite/accept',
```
though, not sure about it

Reply:
It's /cody/invite**s**/accept. I probably would've named it `invite`, but @chrsmith named it this way and I found it difficult to argue for `invite` so just left it as-is. It's definitely a bit confusing that the other path is `/cody/team/manage` with singular "team", which was my naming. We can easily harmonize these later if we want.